src/pages/booklist/containers/BooklistContainer.tsx

```diff
@@ -51,18 +48,11 @@ export function BooklistContainer(): JSX.Element | null {
               .map(({ sections, subject, code }) => {
                 return <TextbookCard subject={subject} code={code} sections={sections} />;
               })
+          ) : textbooks.textbookInfo.filter((textbook) => textbook && textbook.term === term).length <= 0 &&
+            textbooks.textbookInfo.length <= 0 ? (
+            <NotFound item="Unable to find saved courses from your" term={term} timetable />
           ) : (
-            <>
-              <Divider my="4" />
-              <Container alignItems="center" maxW="container.xl">
-                <Heading size="md" color={mode('gray', 'dark.header')}>
-                  Unable to find saved courses or textbooks for{' '}
-                  <Text as="span" color={mode('black', 'white')}>
-                    {getReadableTerm(term)}
-                  </Text>
-                </Heading>
-              </Container>
-            </>
+            <NotFound item="No textbooks found for your saved courses in" term={term} timetable={false} />
```

Comment:
```suggestion
            <NotFound item="No textbooks found for your saved courses in" term={term} />
```
since `timetable` is optional (uses the `?` in the type definition) not passing it will count it as falsey, which is generally a bit more clean